tests/tests_files_forwards.yml

```diff
@@ -51,27 +53,23 @@
 
     - name: Check rsyslog.conf size
       assert:
-        that: rsyslog_conf_line_count.stdout.0|int <= 7
+        that: rsyslog_conf_line_count.stdout.0 | int <= 7
 
     - name: Check file counts in rsyslog.d
       assert:
         that: rsyslog_d_file_count.matched >= 9
 
-    - name: Set files config path
-      set_fact:
-        test_files_conf: "/etc/rsyslog.d/30-output-files.system"
-
     - name: Get the files config stat
       stat:
-        path: "{{ test_files_conf }}"
+        path: "{{ __test_files_conf }}"
       register: stat_output_files_system
 
     - name: Check if the files config exists
       assert:
-        that: stat_output_files_system.stat.exists == True
+        that: stat_output_files_system.stat.exists == true
```

Comment:
why compare a boolean to true? won't it work to just use `stat_output_files_system.stat.exists` ?

Reply:
this applies to other places as well - but please test it first